_posts/2019-10-29-dhis2-and-docker.md

```diff
@@ -0,0 +1,214 @@
+---
+title: DHIS2 with docker
+layout: post
+categories: [blog, docker]
+author: gintare
+---
+
+# DHIS2 with docker
+Docker makes core-teams development and testing efforts easier every day! It allows us to spin up various environments with any DHIS2 version, numerous tomcat versions and different components, such as redis, nginx, in no time. Because we wanted to share docker with community, we made publishing docker images a part of our delivery pipeline. This guide aims to provide some guidelines on how to use docker to quickly set up DHIS2. 
+
+## Prerequisites
+This guide is aimed at people with at least minimal knowledge about docker. If you don't know too much about docker just yet, but think that it might be useful to you, I encourage you to take a look at resources listed in [resources](#Resources) section. 
```

Comment:
I do talk about why it's useful for us in the beginning, so I think there's not much point to repeat that. Also, I don't send them off, I point them to resources to read more :D By minimal knowledge, I mean, that person who knows the concept and I don't need to go back to the basics, explain how docker works and etc. Feel free to update if you want

Reply:
>I do talk about why it's useful for us in the beginning, so I think there's not much point to repeat that.

Fair enough. :+1: 

My point is that vague terms like "minimal" and "not too much" are very hard for the reader to quantify in general, so I try to avoid them. For example, compare:

>This guide is aimed at people with at least minimal knowledge about docker. If you don't know too much about docker just yet [snip]

> This guide requires familiarity with containerization as a concept, running shell commands, and knowledge about how Docker and Docker Compose works. I encourage you to take a look at the [resources](#Resources) section below to get up to speed.

This may not feel right for your writing style however, so feel free to drop it. :smiley:

Reply:
I went with you suggestion, sounds nice :D